_rules/SC4-1-1-unique-attrs.md

```diff
@@ -0,0 +1,135 @@
+---
+name: attributes are not duplicated
+rule_type: atomic
+description: |
+  This rule checks that HTML and SVG starting tags do not contain duplicated attributes.
+
+success_criterion:
+- 4.1.1 # Parsing
+
+test_aspects:
+- Source code
+
+authors:
+- Wilco Fiers
+- Emma Pratt Richens
+---
+
+
+## Applicability
+
+The rule applies to any [starting tag](https://www.w3.org/TR/html5/syntax.html#start-tags) in an HTML or SVG file.
```

Comment:
While HTML and SVG _documents_ can certainly be stored as _files_ on a _file system_, there's also other ways to store/generate such documents. For that reason, I'd use the term "document" instead of "file".